.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,34 @@
+---
+name: Bug Report
+about: Report a bug. Not for asking general questions - see below.
+
+---
+
+<!--
+
+Thanks for opening an issue! To help the Numba team handle your information
```

Comment:
```suggestion
Thanks for opening an issue! To help the llvmlite team handle your information
```
Should this be Numba or llvmlite? In practice, at present, the Numba engineers are a superset of llvmlite engineers.

Reply:
I left it as Numba with my reasoning being that it's the Numba engineers who look after llvmlite - it doesn't feel to me that there's a separate team at all.

Reply:
Indeed, the reality is that is it the same engineers, agree with just leaving it! Thanks!